demo.py

```diff
@@ -99,17 +99,30 @@ def print_sep(char='-'):
             print 'Checking status of authentication request...'
             
             try:
-                request_status = api.get_authentication_status(request_id)
+                auth_request_status = api.get_authentication_request_by_id(auth_request_id)
             except ToopherApiError, e:
                 print 'Could not check authentication status (reason: %s)' % e
                 continue
             
-            if request_status.pending:
+            if auth_request_status.pending:
                 print 'The authentication request has not received a response from the phone yet.'
             else:
-                automation = 'automatically ' if request_status.automated else ''
-                result = 'granted' if request_status.granted else 'denied'
+                automation = 'automatically ' if auth_request_status.automated else ''
+                result = 'granted' if auth_request_status.granted else 'denied'
                 print 'The request was ' + automation + result + "!"
                 break
             
         raw_input('Press return to authenticate again, or Ctrl-C to exit')
+
+def demo():
+    api = initialize_api()
+    pairing = pair_device_with_toopher(api)
+    if pairing.enabled:
+       authenticate_with_toopher(api, pairing)
+
+if __name__ == '__main__':
+    print_sep('=')
+    print 'Library Usage Demo'
+    print_sep('=')
```

Comment:
I find this really ugly--can you find a better way? We could rename to `print_text_with_underline('Toopher Library Demo')` or take out the separator lines or whatever you can think of.